internal/otel/config/helpers/exporters/otlp_exporter.go

```diff
@@ -19,6 +20,7 @@ import (
 const (
 	// otlpHTTPExporterName is the component.ID value used by the otlphttp exporter.
 	otlpHTTPExporterName = "otlphttp"
+	otlpGRPCExporterName = "otlp"
```

Comment:
do we want this to be `otlpgrpc` ? to have it like the http version, or is it that this is a preferred/ default

Reply:
This has to match the component.ID in the exporter